mujoco_warp/test_data/__init__.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2025 The Newton Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Test fixtures for MJWarp."""
+
+from typing import Any, Optional, Sequence, Tuple, Union
+
+import mujoco
+import numpy as np
+from etils import epath
+
+import mujoco_warp as mjw
+from mujoco_warp._src.io import override_model
+
+
+def fixture(
+  path: Optional[str] = None,
+  xml: Optional[str] = None,
+  keyframe: Optional[Union[int, str]] = None,
+  qpos_noise: Optional[float] = None,
+  qvel_noise: Optional[float] = None,
+  ctrl_noise: Optional[float] = None,
+  qfrc_noise: Optional[float] = None,
+  xfrc_noise: Optional[float] = None,
+  mocap_noise: Optional[float] = None,
+  overrides: Union[dict[str, Any], Sequence[str]] = tuple(),
+) -> Tuple[mujoco.MjModel, mujoco.MjData, mjw.Model, mjw.Data]:
+  """Loads MuJoCo MjModel / MjData and corresponding mjw.Model / mjw.Data.
+
+  Args:
+    path: path to the XML file to load, relative to the `test_data` root
```

Comment:
nit: maybe add a period at the end of the description since the other variable descriptions end with a period.